README.md

```diff
@@ -8,125 +8,98 @@ Please use the [covid](https://github.com/epidemics/covid/issues/new/choose) rep
+We are using [luigi](https://luigi.readthedocs.io/en/stable/index.html) as the workflow framework. This
+readme doesn't include description of how to use `luigi` so please refer to the project documentation
+to understand how to tweak this.
 
-   `./do web-export out/batch-out.hdf5 data/sources/estimates-JK-2020-04-15.csv`
-
-8. Export the generated folder to web! Optionally, set a channel for testing first.
-   `./do web-upload -d out/export-2020-04-03T02:03:28.991629+00:00 ttest28`
-
-Alternatively, steps 7 and 8 can be combined:
-
-`./do web-export out/batch-2020-04-16T03:54:52.910001+00:00.hdf5 data/sources/estimates-JK-2020-04-15.csv --upload ttest28`
-
-### Running pipeline with workflow commands
-
-An alternative way to run the pipeline is as follows:
+## Running the pipeline
+### Example with faked data
+This example skips the `UpdateForetold` and `ExtractSimulationsResults` task by providing their output.
+In reality, you want to actually run gleamviz in between and provide Foretold channel to get data via API.
+This by default uses data in `data/inputs` and exports data to `data/outputs/example`.
+```
+# `poetry shell`  # if you haven't done already
+./luigi WebExport \
+    --export-name test-export \
+    --UpdateForetold--output-directory data-dir/inputs/fixtures \
+    --ExtractSimulationsResults--output-directory data-dir/inputs/fixtures \
+    --ExtractSimulationsResults-single-result this-is-now-ignored
+```
 
-1. Update Johns Hopkins and Foretold data, generate batch file from estimates and basic Gleam XML definition and export Gleam simulation XML files to Gleamviz (not while gleamviz is running!):
-   `./do workflow-prepare-gleam -D 2020-04-15 -c JK default.xml estimates-2020-04-15.csv`
+After the pipeline finishes, you should see the results in `data-dir/outputs/example/`
 
-2. Start gleamviz. You should see the new simulations loaded. Run all of them and "Retrieve results" (do not export manually). Exit gleamviz.
+### The usual flow
+You provide all file inputs, foretold_channel and parameters, tweak configs to your liking and then:
 
-3. Import the gleamviz results into the HDF batch file, generate web export and export the generated folder to web (Gleamviz must be stopped before that.) After this succeeds, you may delete the simulations from gleamviz.
-   `./do workflow-gleam-to-web out/batch-2020-04-16T03:54:52.910001+00:00.hdf5 data/sources/estimates-JK-2020-04-15.csv ttest28`
+1. `./luigi GenerateSimulationDefinitions`
+2. run gleamviz with the simulations created above, retrieve results via it's UI, close it
+3. ```
+    ./luigi WebExport \
+    --export-name my-export \
+    --ExtractSimulationResults-single-result ~/GLEAMviz/data/simulations/82131231323.ghv5/results.h5
+  ```
 
-### Gleam Batch file
+### Actually using it
+1. add `foretold_channel` in `luigi.cfg` to `[UpdateForetold]` section (you can get it from others on slack)
```

Comment:
what is foretold_channel?

Reply:
https://github.com/epidemics/epimodel/pull/60/commits/c830a47992acc052190a393886f00c98b2e6e433